types/run-sequence/package.json

```diff
@@ -1,12 +1,12 @@
 {
     "private": true,
     "name": "@types/run-sequence",
-    "version": "0.0.9999",
+    "version": "2.2.9999",
     "projects": [
         "https://github.com/OverZealous/run-sequence"
     ],
     "dependencies": {
-        "@types/gulp": "*",
+        "@types/gulp": "^3.8",
```

Comment:
Is it `^3.8` or `>=3.8` v3 is old...

Reply:
Done, and I have updated the PR description.

> - Here `@types/undertaker` is installed to reference the `gulp.task` callback type.
>     - v4 is using this package for `gulp.task` callback type.
>     - But v3 **is not**, and in fact its `gulp.task` callback type conflict with the v4 one.
>     - Directly using the type from `@types/undertaker` help resolve type issue in both major version.
> - Since `gulp.TaskCallback` is used and the [original package](https://github.com/OverZealous/run-sequence/blob/master/package.json#L31) expects `"gulp": "^3.9.1",`, the `package.json` in this PR is hence modified accordingly, to use `>=3.8`.